doc/reference/common-weaknesses/exposed-keys.rst

```diff
@@ -0,0 +1,15 @@
+Passwords or Keys may be exposed on the blockchain
+==================================================
+
+If information which is supposed to remain private, such as a password for example, is stored in UTXO data or redeemers, then it may be read by anyone with access to the transaction pool or the blockchain. When secret data is embedded in larger data structures, then it is easy to forget which values may contain secrets, and thus to leak them inadvertently.
+
+Risks
+~~~~~
+
+Making secrets publicly visible may allow arbitrary contract execution leading to loss of funds.
```

Comment:
There are a few risks sections that basically just say "this could cause anything to happen, totally depends on your application". Not sure what they do in CWE-land, they must have the same problem. (What are the risks of someone gaining elevated privileges to their user account? Basically anything - depends on the app)